google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/standard/AppEngineStandardStage.java

```diff
@@ -68,11 +72,22 @@ public void onOutputLine(String line) {
 
     // TODO determine the default set of flags we want to set for AE standard staging
     DefaultStageStandardConfiguration stageConfig = new DefaultStageStandardConfiguration();
-    stageConfig.setEnableJarSplitting(true);
```

Comment:
@etanshaul Do we want to set the default value of enable_jar_splitting to true, since at some point we started setting that value to true internally?

Reply:
I think it was pointed out (by @chanseokoh?) that changing this default value could cause some changes on how the project runs.

@patflynn do you know if we have any guidance on what defaults we should use, or is it up to us to come up with sensible defaults?

Reply:
Lets cross reference with the doc and gcloud default behavior to determine what defaults to set for these.